jax/_src/scipy/stats/bernoulli.py

```diff
@@ -36,3 +36,29 @@ def logpmf(k: ArrayLike, p: ArrayLike, loc: ArrayLike = 0) -> Array:
 @_wraps(osp_stats.bernoulli.pmf, update_doc=False)
 def pmf(k: ArrayLike, p: ArrayLike, loc: ArrayLike = 0) -> Array:
   return jnp.exp(logpmf(k, p, loc))
+
+@_wraps(osp_stats.bernoulli.cdf, update_doc=False)
+def cdf(k: ArrayLike, p: ArrayLike) -> Array:
+  k, p = jnp._promote_args_inexact('bernoulli.cdf', k, p)
+  zero, one = _lax_const(k, 0), _lax_const(k, 1)
+  conds = [
+    jnp.isnan(k),
+    lax.lt(k, zero),
+    jnp.logical_and(lax.ge(k, zero), lax.lt(k, one)),
+    lax.ge(k, one)
+    ]
+  vals = [jnp.nan, 0, 1-p, 1]
```

Comment:
We should ensure the output dtype matches the input dtype. you can do this by using `lax._const` for each of the entries in this list